libr/core/cmd_print.c

```diff
@@ -1839,6 +1839,15 @@ static void disasm_strings(RCore *core, const char *input, RAnalFunction *fcn) {
 		}
 		if (addr != UT64_MAX) {
 			const char *str = NULL;
+			char *comment = NULL;
+			bool show_comments = r_config_get_i (core->config, "asm.comments");
+			comment = r_core_anal_get_comments (core, addr);
+			if (str && show_comments) {
+				r_cons_printf ("%s0x%08"PFMT64x, use_color? Color_GREEN: "", addr);
```

Comment:
didn't mean to push sorry.
trying to figure out how to use the palete. will fix asap